### PR TITLE
Task/question list view model dima

### PR DESCRIPTION
**Pull request by Promah**

### CODE REVIEW

**Comment by Promah** [comment]
something like that?

File: app/src/main/java/ua/onseo/ds/ui/question_list/ServerResponseMapper.kt [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package ua.onseo.ds.ui.question_list
+
+import ua.onseo.ds.repository.model.QuizList
+import ua.onseo.ds.repository.model.QuizListItem
+import ua.onseo.ds.repository.model.ServerResponse
+
+class ServerResponseMapper(val response: ServerResponse<*>) {
```
Comment (YPoudanenOnSeo) [inline app/src/main/java/ua/onseo/ds/ui/question_list/ServerResponseMapper.kt]: Make concrete ServerResponse instead of *

Comment (Promah) [reply app/src/main/java/ua/onseo/ds/ui/question_list/ServerResponseMapper.kt]: ServerResponseMapper changed to concrete QuizListItemsMapper

File: app/src/main/java/ua/onseo/ds/utils/SingleLiveEvent.kt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import android.arch.lifecycle.LifecycleOwner
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.Observer
+import android.support.annotation.MainThread
+import android.support.annotation.Nullable
+import android.util.Log
+import java.util.concurrent.atomic.AtomicBoolean
+
+
+class SingleLiveEvent<T> : MutableLiveData<T>() {
```
Comment (YPoudanenOnSeo) [inline app/src/main/java/ua/onseo/ds/utils/SingleLiveEvent.kt]: What the purpose to make your own LiveData?

**Pull request merged**